isites_migration/views.py

```diff
@@ -31,6 +31,14 @@ def index(request):
             term=term,
             canvas_course_id=canvas_course_id
         )
+        try:
+            ci = CourseInstance.objects.get(course_instance_id=course_instance_id)
+            school = ci.course.school_id
+        except CourseInstance.DoesNotExist as e:
+            school = ''
```

Comment:
Do we want to fall back to looking up the course instance by canvas course id here?